lookout/style/format/descriptions.py

```diff
@@ -8,26 +8,12 @@
 from numpy import flatnonzero, floating, ndarray
 
 from lookout.style.format.feature_extractor import FeatureExtractor, FEATURES_MAX, FEATURES_MIN
-from lookout.style.format.feature_utils import CLASS_INDEX, CLASSES, CLS_DOUBLE_QUOTE, \
-    CLS_NEWLINE, CLS_NOOP, CLS_SINGLE_QUOTE, CLS_SPACE, CLS_SPACE_DEC, CLS_SPACE_INC, CLS_TAB, \
-    CLS_TAB_DEC, CLS_TAB_INC, VirtualNode
+from lookout.style.format.feature_utils import (
+    CLASS_INDEX, CLASS_REPRESENTATIONS, CLS_NEWLINE, CLS_NOOP)
+from lookout.style.format.feature_utils import VirtualNode
 from lookout.style.format.features import BagFeature, CategoricalFeature, OrdinalFeature
 from lookout.style.format.rules import Rule
 
-_CLASS_REPRESENTATIONS_MAPPING = {
-    CLS_DOUBLE_QUOTE: '"',
-    CLS_NEWLINE: "⏎",
-    CLS_NOOP: "∅",
-    CLS_SINGLE_QUOTE: "'",
-    CLS_SPACE: "␣",
-    CLS_SPACE_DEC: "␣⁻",
-    CLS_SPACE_INC: "␣⁺",
-    CLS_TAB: "⇥",
-    CLS_TAB_DEC: "⇥⁻",
-    CLS_TAB_INC: "⇥⁺",
-}
-CLASS_REPRESENTATIONS = [_CLASS_REPRESENTATIONS_MAPPING[cls] for cls in CLASSES]
-del _CLASS_REPRESENTATIONS_MAPPING
 
 CLASS_PRINTABLES = copy(CLASS_REPRESENTATIONS)
```

Comment:
If we move REPRESENTATIONS we should also move PRINTABLES

Reply:
agree.